radio/src/targets/common/arm/stm32/audio_dac_driver.cpp

```diff
@@ -60,7 +60,7 @@ void dacInit()
   GPIO_InitStructure.GPIO_OType = GPIO_OType_PP;
   GPIO_InitStructure.GPIO_PuPd = GPIO_PuPd_UP;
   GPIO_Init(AUDIO_MUTE_GPIO, &GPIO_InitStructure);
-  GPIO_SetBits(AUDIO_MUTE_GPIO, AUDIO_MUTE_GPIO_PIN);
+  setMutePin(AUDIO_MUTE_GPIO,AUDIO_MUTE_GPIO_PIN);
```

Comment:
This is a common header, used by colorlcd and B&W radios... however you've only defined these `setMutePin, resetMutePin, readMutePinLevel` you've only defined them for the taranis in `board.cpp`. As to what the preferred resolution is, this is up to @raphaelcoeffic to decide. However, this is what is breaking the build checks for this PR currently as far as I can tell.

Reply:
Yes, this part is an open problem we were having... Hope we could get some feedback for a better solution

Reply:
The better way would be to provide these functions in the driver itself, and use a specific `define` called `INVERTED_MUTE_PIN` in `hal.h` to express that the logic is inverted.

In `targets/taranis/hal.h`, you would then define `INVERTED_MUTE_PIN` only for your target.